SPECS/kernel/config

```diff
@@ -2314,7 +2314,7 @@ CONFIG_TAP=m
 CONFIG_VETH=y
 CONFIG_VIRTIO_NET=y
 # CONFIG_NLMON is not set
-# CONFIG_NET_VRF is not set
+CONFIG_NET_VRF=m
```

Comment:
Double checking here:
Since we are building it as a module, would we want this in the `initrd` as well or we use this once the system is fully up ?

Reply:
Hmmm good thought. I added it to the drivers argument list for dracut